PokemonAPI/PokemonAPI.BusinessLayer/Validator/TrainerValidator.cs

```diff
@@ -0,0 +1,48 @@
+﻿using FluentValidation;
+using PokemonAPI.DomainLayer.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PokemonAPI.BusinessLayer.Validator
+{
+    public class TrainerValidator : AbstractValidator<TrainerDTO>
+    {
+		public TrainerValidator()
+		{
+			RuleFor(x => x.FullName).Cascade(CascadeMode.StopOnFirstFailure).NotEmpty().WithMessage("Trainer full name is required")
```

Comment:
break this it will be a bit more readable.

Can we get this to return multiple messages somehow?
Like in pokemonvalidation 
So it doesn't fail instantly on first error